server/views/licenceConditions/standard.pug

```diff
@@ -3,6 +3,7 @@ extends ../layout
 block content
 
   - var additionalConditionsValue = data.additionalConditionsRequired || null;
+  - var licenceConditions = licenceWithReconstructedConditions.licenceConditions || {}
```

Comment:
this seems a bit confusing, why do we have reference to 2 things here - `conditions` and `standardConditons`?
Can we simplify this?

Reply:
it seems like the variable called `conditions` now needs a better name?

Reply:
but also do you even need to pass it in? does licenceConditions also include it?

Reply:
the standard conditions were already there. 
This ticket was to add the additional, pss and bespoke conditions

Reply:
amended